docs/reference/common-configs.md

```diff
@@ -0,0 +1,104 @@
+<!--[metadata]>
++++
+title = "Common Server and Signer Configurations"
+description = "Configuring the notary client, server and signer."
+keywords = ["docker, notary, notary-client, notary-server, notary server, notary-signer, notary signer"]
+[menu.main]
+parent="mn_notary_config"
+weight=5
++++
+<![end-metadata]-->
+
+# Configure sections common to Notary server and signer
+
+This document is for power users of the [notary client](../advanced_usage.md),
```

Comment:
There's some inconsistency here. If this common config is just for server and signer we shouldn't mention the client.